src/lib.rs

```diff
@@ -351,6 +337,21 @@ impl Default for Compressor {
     }
 }
 
+impl Compressor {
+    #[inline]
+    fn get_twobyte(&self, code: u16) -> CodeMeta {
+        unsafe { *self.codes_twobyte.get_unchecked(code as usize) }
+    }
```

Comment:
these probably don't actually matter much but i was just trying to avoid bounds checks wherever possible